src/components/Sidebar.js

```diff
@@ -1,13 +1,72 @@
-import { Link } from 'react-router-dom';
+import { useState } from 'react';
+import { Link, useLocation } from 'react-router-dom';
+import { Icon } from '@iconify/react';
 
 export default function Sidebar() {
+  const [open, setOpen] = useState(true);
+
   return (
-    <div className="w-60 p-4 flex-shrink-0 shadow-xl">
-      <h1 className="font-semibold text-2xl mb-8">Coverage Checker</h1>
-      <nav className="flex flex-col gap-2">
-        <Link to="/file-upload">Upload Files</Link>
-        <Link to="/">mCODE Coverage</Link>
-      </nav>
+    <div className={`flex flex-col ${open ? 'w-60' : 'w-20'} p-4 h-screen sticky top-0 flex-shrink-0 shadow-xl`}>
+      {/* sidebar header */}
+      <div className="flex flex-row">
+        <h1 className={`${open ? '' : 'hidden'} font-semibold text-xl mb-8`}>Coverage Checker</h1>
+        <Icon
+          icon="akar-icons:chevron-down"
+          className="ml-2"
+          height="24"
+          rotate="90deg"
+          vFlip={!open}
+          onClick={() => setOpen(!open)}
+        />
+      </div>
+      {/* sidebar body */}
+      <div className="flex-1">
+        {/* upper section */}
+        <div className="h-3/5 flex flex-col gap-5">
+          <h2 className="font-semibold text-base mb-2">{open ? 'Navigation' : 'Nav'}</h2>
+          <Link
+            to="/file-upload"
+            className={`flex flex-row rounded-sidebarButton ${
+              useLocation().pathname === '/file-upload' ? 'shadow-sidebarButton' : ''
+            }`}
+          >
+            <Icon icon="bi:folder" height="24" className="ml-2" />
+            <p className={`${open ? 'pl-2' : 'hidden'}`}>Upload Files</p>
+          </Link>
+          <Link
+            to="/"
+            className={`flex flex-row rounded-sidebarButton ${
+              useLocation().pathname === '/' ? 'shadow-sidebarButton' : ''
+            }`}
+          >
+            <Icon icon="fluent:database-20-filled" height="24" className="ml-2" />
+            <p className={`${open ? 'pl-2' : 'hidden'}`}>mCODE Coverage</p>
+          </Link>
+        </div>
+        {/* lower section */}
+        <div className="flex flex-col gap-5">
+          <hr className="h-px bg-gray-300 border-0" />
+          <h2 className="font-semibold text-base mb-2">Help</h2>
```

Comment:
Let's replace `Help` with `About`, since we're really not pointing them to any help or documentation on how to use the tool. 

Also, let's use the same new font-color from Nav, `heading-grey` over here